todo/src/components/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { toggleTodo } from '../actions/todoActions';
+import { connect } from 'react-redux';
+
+class Todo extends Component {
+
+    toggleTodo = (e, index) => {
+        e.preventDefault();
+        this.props.toggleTodo(index)
+        console.log('toggle')
+    };
+
+    render() {
+        return (
+            <ul>
+                <li onClick={this.toggleTodo}>
```

Comment:
```suggestion
                <li onClick={e => this.toggleTodo(e, this.props.index)}>
```
Since index was passed as a prop you have to access it like this and when you call your helper function you need to pass it event and index from props since it expects those arguements